src/routes/v2/authenticated/sites.ts

```diff
@@ -188,6 +195,40 @@ export class SitesRouter {
     return res.status(404).json({ message: "Unable to get staging status" })
   }
 
+  getLinkCheckerStatus: RequestHandler<
+    { siteName: string },
+    BrokenLinkErrorDto | ResponseErrorBody,
+    never,
+    never,
+    { userWithSiteSessionData: UserWithSiteSessionData }
+  > = async (req, res) => {
+    const { userWithSiteSessionData } = res.locals
+    const result = await this.repoCheckerService.readRepoErrors(
+      userWithSiteSessionData.siteName
+    )
+    if (result.isOk()) {
+      return res.status(200).json(result.value)
+    }
+    return res.status(404).json({ status: "error" })
```

Comment:
status could be better phrased e.g. error: no log files found

Reply:
hmm wait this the shape of the error dto. for now our fe only has one error state, which the frontend takes care of, so am inclined to keep it simple for the initial rollout.

additionally, this could have multiple failure modes, eg file could not be parsed
but it is no meaningful to return these varying error states since from the user pov, there is only one way to recover, which is to retry